server/schedulers/evict_leader.go

```diff
@@ -354,7 +354,7 @@ func (handler *evictLeaderHandler) DeleteConfig(w http.ResponseWriter, r *http.R
 		return
 	}
 
-	handler.rd.JSON(w, http.StatusNotFound, ErrScheduleConfigNotExist)
+	handler.rd.JSON(w, http.StatusNotFound, errs.ErrScheduleConfigNotExist)
```

Comment:
I think in the first stage, we don't need to change the output of API?

Reply:
Do you mean to keep the error in schedule.util?

Reply:
What do you think?

Reply:
If so, when will we merge them?

Reply:
I'm not sure, but for now, we only need to unify the output of err log.

Reply:
done